aws-lambda-java-runtime-interface-client/scripts/test-rie.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(dirname "$SCRIPT_DIR")"
+SERIALIZATION_ROOT="$(dirname "$PROJECT_ROOT")/aws-lambda-java-serialization"
+
+if ! ls "$PROJECT_ROOT"/target/aws-lambda-java-runtime-interface-client-*.jar >/dev/null 2>&1; then
+  echo "RIC jar not found. Please build the project first with 'mvn package'."
+  exit 1
+fi
+
+IMAGE_TAG="java-ric-rie-test"
+
+HANDLER="${1:-EchoHandler::handleRequest}"
+
+echo "Starting RIE test setup for Java..."
+
+# Download required dependencies if not present
+if ! ls "$PROJECT_ROOT"/target/aws-lambda-java-core-*.jar >/dev/null 2>&1; then
+  echo "Downloading aws-lambda-java-core..."
+  (cd "$PROJECT_ROOT" && mvn dependency:copy -Dartifact=com.amazonaws:aws-lambda-java-core:RELEASE -DoutputDirectory=target)
```

Comment:
I think this is downloading the artifact from maven central instead of using the built version?
```
[INFO] --- dependency:3.6.1:copy (default-cli) @ aws-lambda-java-runtime-interface-client ---
[INFO] Configured Artifact: com.amazonaws:aws-lambda-java-core:RELEASE:jar
Downloading from central: https://repo.maven.apache.org/maven2/com/amazonaws/aws-lambda-java-core/maven-metadata.xml
```